include/cache_policy.hpp

```diff
@@ -64,7 +64,7 @@ class NoCachePolicy : public ICachePolicy<Key>
         key_storage.emplace(key);
     }
 
-    void Touch(const Key &key) noexcept override
+    void Touch(__attribute__((unused)) const Key &key) noexcept override
```

Comment:
Unfortunately, MSVC does not support `__attribute__` stuff. Instead, to keep things portable, we should use something like this:

```cpp
void Touch(const Key &key) noexcept override
{
    (void)key;
}
```

Reply:
The suggested approach works for me in clang50. Updated PR.